dotnet/samples/Concepts/Functions/MethodFunctions_Yaml.cs

```diff
@@ -0,0 +1,68 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Reflection;
+using Microsoft.SemanticKernel;
+
+namespace Functions;
+
+public class MethodFunctions_Yaml(ITestOutputHelper output) : BaseTest(output)
+{
+    private const string FunctionConfig = """
+        name: ValidateTaskId
+        description: Validate a task id.
+        input_variables:
+          - name: kernel
+            description: Kernel instance.
+          - name: taskId
+            description: Task identifier.
+            is_required: true
+        output_variable:
+          description: String indicating whether or not the task id is valid.
+        """;
+
+    /// <summary>
+    /// This example create a plugin and uses a separate configuration file for the function metadata.
```

Comment:
Is this scenario common?  Would it make sense for us to just provide developers a helper to combine these?  It seems like there is typically only one way to combine these, since it's our CreateFromMethod function and our Yaml config anyway.